app/MindWork AI Studio/Chat/ChatThread.cs

```diff
@@ -69,6 +74,8 @@ public sealed record ChatThread
     /// The content blocks of the chat thread.
     /// </summary>
     public List<ContentBlock> Blocks { get; init; } = [];
+    
+    private bool allowProfile = true;
```

Comment:
A private variable? Where is it set then? Or does the chat template have or know the status of whether a profile may be used?